### PR TITLE
Added attributes to ignored list and fixed a11y-variant in notice

### PR DESCRIPTION
**Pull request by austingardner**
## Description
Fixes issues Melissa raised that were assigned to me.

## References
https://github.com/eBay/ebayui-core/issues/1285
https://github.com/eBay/ebayui-core/issues/1281
https://github.com/eBay/ebayui-core/issues/1280
https://github.com/eBay/ebayui-core/issues/1279

### CODE REVIEW

File: src/components/ebay-section-notice/test/test.server.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ describe('section-notice', () => {
     it('renders with status', async() => {
         const input = mock.Section_Info;
         const { getAllByLabelText, getByText } = await render(template, input);
-        const status = getAllByLabelText(input.a11yText)[1].parentElement;
+        const status = getAllByLabelText(input.a11yText)[0].parentElement;
```
Comment (agliga) [inline src/components/ebay-section-notice/test/test.server.js:12]: Can we make sure now that `getAllByLabelText` returns a list and not just one element?
If it returns a list this is fine, but if it returns only one, we can use `getByLabelText` instead

File: src/components/ebay-inline-notice/index.marko [hunk 2]
```diff
@@ -8,4 +8,5 @@ $ var status = input.status || "attention";
     root="div"
     header-root="span"
     main-root="span"
+    a11y-text=input.a11yText
```
Comment (agliga) [inline src/components/ebay-inline-notice/index.marko:11]: This is not needed, `a11yText` should get passed through from the `...input`. 
If it's not, then we need to add `a11y-text` in marko.json
Check this again

**Review by agliga — COMMENTED** [review]
Two things
* Change the title for the pull request
* We shouldn't have any `yarn.lock` changes
Could of other minor issues to check too

Also don't have to change this now, but usually for combined PRs try to make each commit to resolve and link to each issue. Once we merge, we can merge without squashing since there is a commit associated for each fix. 
However for now leave it as is, but for future we can do that.

File: src/components/ebay-page-notice/test/test.server.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ describe('page-notice', () => {
     it('renders with defaults', async() => {
         const input = mock.Page;
         const { getAllByLabelText, getByText } = await render(template, input);
-        const status = getAllByLabelText(input.a11yText)[1].parentElement;
+        const status = getAllByLabelText(input.a11yText)[0].parentElement;
```
Comment (agliga) [inline src/components/ebay-page-notice/test/test.server.js:13]: This one didn't work with `getByLabelText`?

Comment (austingardner) [reply src/components/ebay-page-notice/test/test.server.js:13]: ah dang missed that one. no it should work. one sec

**Review by agliga — APPROVED** [review]
Lgtm
Just change the title to describe the changes. I wouldnt keep 6.0 bugfixes but rather like
Added attributes to ignored list and fixed a11y-variant in notice

**Pull request merged**